src/App/Template/AbstractTemplate.php

```diff
@@ -16,11 +16,88 @@ abstract class AbstractTemplate extends AbstractWebTemplate
      */
     const ASSETS_VERSION = '1.0';
 
+    const CORE_TEMPLATE_ACTIONS = [
+        'siteName',
+        'google',
+        'typekit',
+        'htmlAttrStructure',
+        'htmlAttr',
+        'assetsVersion',
+        'copyright',
+        'copyrightName',
+        'copyrightYear',
+        'criticalCss',
+        'templateName',
+        'title',
+        'currentLocale',
+        'currentUrl',
+        'currentLanguage',
+        'metaTitle',
+        'metaDescription',
+        // 'metaImage',
+        'opengraphType',
+        // 'opengraphImage',
+        'seoMetadata',
+        'baseUrl',
+        'availableLanguages',
+        'hasSeoMetadata',
+        'documentTitle',
+        'hasAlternateTranslations',
+        'alternateTranslations',
+    ];
+
+    /**
+     * Gets the data keys on this entity.
+     * @see
+     *
+     * @return array
+     */
+    public function keys($keys = null)
```

Comment:
The `keys()` method does not accept parameters.

```suggestion
    /**
     * Retrieves the view data keys on this template.
     *
     * @override Builds list of data keys dynamically.
     *
     * @return string[]
     */
    public function keys()
```